frontend/src/views/ddahs/index.tsx

```diff
@@ -34,6 +40,43 @@ export function AdminDdahsView(): React.ReactNode {
         selectedDdahIds.includes(ddah.id)
     );
 
+    const [
+        ddahDeletionConfirmationVisible,
+        setDdahDeletionConfirmationVisible,
+    ] = React.useState(false);
+    const [ddahBrief, setDdahBrief] = React.useState("");
+
+    function confirmDDAHDeletion() {
+        if (selectedDdahs?.length > 1) {
+            let multipleOfferWithdrawBrief = "";
+
+            selectedDdahs.forEach((selectedDdah) => {
+                multipleOfferWithdrawBrief += `${
+                    selectedDdah.assignment.applicant.first_name
+                } ${selectedDdah.assignment.applicant.last_name}: ${
+                    selectedDdah.assignment.position.position_code
+                } ${selectedDdah.assignment.position.position_title} -- ${
+                    selectedDdah.status ? selectedDdah.status : "unsent"
+                } (total hours: ${selectedDdah.total_hours})\n`;
```

Comment:
This is confusing. Use if statements. Also, make this a JSX object instead of a string.